packages/mobile/src/account/selectors.ts

```diff
@@ -6,3 +6,5 @@ export const getPaymentRequests = (state: RootState) => {
     (p) => p.status === PaymentRequestStatuses.REQUESTED
   )
 }
+
+export const currentPhoneNumber = (state: RootState) => state.account.e164PhoneNumber
```

Comment:
We already have a selector for this: https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/account/reducer.ts#L122